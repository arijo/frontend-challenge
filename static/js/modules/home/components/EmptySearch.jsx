import React from 'react';
import {Col, Row} from 'react-grid-system';
import style from "../../../../css/modules/home/Home.scss";

const EmptySearch = () => {
    return (
        <div>
            <Row>
                <Col sm={12}>
                    <div className={style.illustrationEmpty}></div>
                </Col>
            </Row>
            <Row>
                <Col sm={12}>
                    <div className={style.marketingCopy}>
                        <div className={style.question}>Don&apos;t know what to search?</div>
                        <div className={style.proposal}>Here&apos;s an offer you can&apos;t refuse</div>
                    </div>
                </Col>
            </Row>
        </div>
    )
}

EmptySearch.displayName = 'EmptySearch';

export default EmptySearch;