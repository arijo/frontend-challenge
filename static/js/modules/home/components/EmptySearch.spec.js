import EmptySearch from './EmptySearch.jsx';

it('renders correctly', () => {
    const wrapper = shallow(
        <EmptySearch/>
    );
    expect(wrapper).toMatchSnapshot();
})