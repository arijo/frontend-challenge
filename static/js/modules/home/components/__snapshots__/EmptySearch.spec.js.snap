// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <Row
    align="normal"
    component="div"
    debug={false}
    justify="start"
    nogutter={false}
    style={Object {}}
  >
    <Col
      component="div"
      debug={false}
      lg={null}
      md={null}
      offset={Object {}}
      pull={Object {}}
      push={Object {}}
      sm={12}
      style={Object {}}
      xl={null}
      xs={null}
    >
      <div
        className="illustrationEmpty"
      />
    </Col>
  </Row>
  <Row
    align="normal"
    component="div"
    debug={false}
    justify="start"
    nogutter={false}
    style={Object {}}
  >
    <Col
      component="div"
      debug={false}
      lg={null}
      md={null}
      offset={Object {}}
      pull={Object {}}
      push={Object {}}
      sm={12}
      style={Object {}}
      xl={null}
      xs={null}
    >
      <div
        className="marketingCopy"
      >
        <div
          className="question"
        >
          Don't know what to search?
        </div>
        <div
          className="proposal"
        >
          Here's an offer you can't refuse
        </div>
      </div>
    </Col>
  </Row>
</div>
`;
